src/core/src/service_interfaces/LifecycleManager.py

```diff
@@ -22,7 +22,7 @@
 
 
 class LifecycleManager(object):
-    """Class for managing the core code's lifecycle within the extension wrapper"""
+    """ Parent class for LifecycleManagers of Azure and ARC ( auto assessment ), manages lifecycle within the extension wrapper ~ """
 
```

Comment:
Refactor test classes to match - Test_LifecycleManager should become Test_LifecycleManagerAzure -- references must also change similarly.